app/assets/javascripts/govuk-admin-template/modules/track_button_press.js

```diff
@@ -0,0 +1,16 @@
+(function(Modules) {
+  "use strict";
+
+  Modules.TrackButtonPress = function() {
+    var that = this;
+    that.start = function(element) {
+      element.find(".js-track").on("click", function() {
```

Comment:
For legibility, could the anonymous function be moved into a named one elsewhere. This reduces nesting and makes it clearer what will happen on click.